VAULT-31525 Remove lowercasing of RADIUS auth. (#28884)
hashicorp/vault: changelog/28884.txt

```release-note:bug

> auth/radius: Fixed and issue where upper case items were returning as lower case.

Reviewer: ```suggestion
auth/radius: Fixed an issue where usernames with upper case characters where not honored
```

We may also want to add a `release-note:change` since this is a behavior change that will also result in changes to how policies are assigned
